modules/cmd_args.py

```diff
@@ -60,8 +60,7 @@
 parser.add_argument("--sub-quad-chunk-threshold", type=int, help="the percentage of VRAM threshold for the sub-quadratic cross-attention layer optimization to use chunking", default=None)
 parser.add_argument("--opt-split-attention-invokeai", action='store_true', help="prefer InvokeAI's cross-attention layer optimization for automatic choice of optimization")
 parser.add_argument("--opt-split-attention-v1", action='store_true', help="prefer older version of split attention optimization for automatic choice of optimization")
-parser.add_argument("--opt-sdp-attention", action='store_true', help="prefer scaled dot product cross-attention layer optimization for automatic choice of optimization; requires PyTorch 2.*")
-parser.add_argument("--opt-sdp-no-mem-attention", action='store_true', help="prefer scaled dot product cross-attention layer optimization without memory efficient attention for automatic choice of optimization, makes image generation deterministic; requires PyTorch 2.*")

```

Comment:
Deprecated settings have been traditionally left in, see e.g. `--disable-opt-split-attention`?

Reply:
I also thought about switching `sdp-no-mem` to `sdp` by default, but because Auto rewrote this part and added it to the interface settings, implementing it would remain misleading and not pretty.

Reply:
It's also problematic if you just keep but "do nothing", not all users will pay attention to the "Apply...sdp...optimization" in the console, they will complain about the slowdown.